src/python/pants/base/build_environment.py

```diff
@@ -60,3 +62,8 @@ def get_git() -> Optional[Git]:
         except GitException as e:
             logger.info(f"Failed to load git repository at {worktree}: {e!r}")
     return _Git
+
+
+def get_global_module_mapping() -> Mapping[str, Iterable[str]]:
```

Comment:
The `pants.base` package is language-agnostic, so this function and the resource should likely live in `src/python/pants/backend/python/dependency_inference` instead.

Reply:
See below for suggested location.